.github/workflows/autobuild.yml

```diff
@@ -127,6 +127,15 @@ jobs:
              uses_codeql:           false
              xcode_version:         11.7
 
+           - config_name:           iOS (artifacts)
+             target_os:             ios
+             building_on_os:        macos-10.15
+             cmd1_prebuild:         "./autobuild/ios/artifacts/autobuild_ios_1_prepare.sh 5.15.2"
+             cmd2_build:            "./autobuild/ios/artifacts/autobuild_ios_2_build.sh"
+             cmd3_postbuild:        "./autobuild/ios/artifacts/autobuild_ios_3_copy_files.sh"
+             uses_codeql:           false
+             xcode_version:         11.7
```

Comment:
This could be 12.1.1 now that #2266 has been merged.

Reply:
Done.